modules/database/src/admin/index.ts

```diff
@@ -351,6 +365,7 @@ export class AdminHandlers {
         {
           path: '/introspection',
           action: ConduitRouteActions.POST,
+          description: `Performs database introspection.`,
```

Comment:
We should ideally be mentioning pending schemas here as we reference them in followup descriptions.
I would suggest something along the lines of:
`Performs database introspection, registering any unknown collections as pending schemas.`